src/scripts/no_recommended/hide_recommended_posts.js

```diff
@@ -1,13 +1,45 @@
-import { buildStyle, filterPostElements } from '../../util/interface.js';
+import { buildStyle, filterPostElements, postSelector } from '../../util/interface.js';
 import { onNewPosts } from '../../util/mutations.js';
 import { timelineObject } from '../../util/react_props.js';
 
 const excludeClass = 'xkit-no-recommended-posts-done';
 const hiddenClass = 'xkit-no-recommended-posts-hidden';
+const unHiddenClass = 'xkit-no-recommended-posts-many';
 const timeline = /\/v2\/timeline\/dashboard/;
 const includeFiltered = true;
 
-const styleElement = buildStyle(`.${hiddenClass} article { display: none; }`);
+const styleElement = buildStyle(`
+.${hiddenClass}:not(.${unHiddenClass}) article {
+  display: none;
+}
+
+:not(.${unHiddenClass}) + .${unHiddenClass}::before {
+  content: 'Too many recommended posts to hide!';
+
+  display: block;
+  padding: 25px 20px;
+  border-radius: 3px;
+  margin-bottom: var(--post-padding);
+
+  background-color: rgba(var(--white-on-dark), 0.13);
+  color: rgba(var(--white-on-dark), 0.6);
```

Comment:
This might be the most annoying change I ever request but... Tumblr never uses 60% opacity colours... only 65%

```suggestion
  color: rgba(var(--white-on-dark), 0.65);
```